servicex/transformer/object_store_manager.py

```diff
@@ -30,16 +30,22 @@
 
 class ObjectStoreManager:
 
-    def __init__(self, url=None, username=None, password=None):
+    def __init__(self, url=None, username=None, password=None, use_https=False):
         from minio import Minio
-
+        if 'MINIO_SECURED' in os.environ:
+            if os.environ['MINIO_SECURED'].lower() == "true":
```

Comment:
This could just be 
```
secure_connection = os.environ['MINIO_SECURED'].lower() == "true"
```

Reply:
Wouldn't that return a KeyError is the MINIO_SECURED variable is not in the environment?

Reply:
Not if you keep the `if 'MINIO_SECURED' in os.environ:` or switch to `os.environ.get("MINIO_SECURED", "false")`

Reply:
Ok - it was painful that this branch caused code coverage to drop. I fixed it...